### PR TITLE
Fix nested with comments

### PR DESCRIPTION
**Pull request by lexich**
Without fix, test crashed

### CODE REVIEW

File: tests/test.js [hunk 1]
```diff
@@ -108,6 +108,31 @@ const nestedDeclarationTests = [
...
+  },
+  {
+    message: 'Sort nested with comments at top',
+    fixture: 'a{&:hover{/*Comment*/ flex: 0; border: 0;}}',
+    expected: 'a{&:hover{ border: 0;/*Comment*/ flex: 0;}}'
+  },
+  {
+    message: 'Sort nested with comments at bottom',
+    fixture: 'a{&:hover{flex: 0;border: 0;/*Comment*/}}',
+    expected: 'a{&:hover{border: 0;/*Comment*/flex: 0;}}'
```
Comment (lexich) [inline tests/test.js]: Maybe this behaviour isn't perfect, but better then crash

**Comment by Siilwyn** [comment]
Hi there Lexich, thank you for finding this bug and submitting this PR. This is why I love open source and what keeps me motivated! :heart: 

I have the feeling it's a bit too much duplicate logic so I took a stab at re-factoring it, please review the PR I made on your branch and share your thoughts: https://github.com/lexich/css-declaration-sorter/pull/1
I've removed the Yarn file as I'd like to keep the tooling as simple as possible, hence I'm also not using any transpiler. Plus I saw some 'duplicate' tests such as the comment ordering so I simplified those to just check if handling comments in a nested block work. :)

**Comment by lexich** [comment]
Yes, code become cleaner. 👍

**Comment by Siilwyn** [comment]
Thanks @lexich! Adding you as a contributor because of this contribution, feel free to accept or reject the invitation.

**Pull request merged**